pkg/mcs/utils/constant/constant.go

```diff
@@ -47,6 +47,11 @@ const (
 	// 0 is reserved for default keyspace with the name "DEFAULT". It's initialized when PD bootstraps
 	// and reserved for users who haven't been assigned keyspace.
 	DefaultKeyspaceID = uint32(0)
+
+	// SystemKeyspaceID is the system keyspace ID.
+	SystemKeyspaceID = uint32(1)
```

Comment:
We still use 1 for system keyspace?

Reply:
It may be more suitable to use uint32.Max - 1024 or uint32.Max - 1  or uint32.Max cc @D3Hunter @JmPotato